### PR TITLE
BEAM-1660 withCoder() error in JdbcIO JavaDoc example

### PR DESCRIPTION
**Pull request by XuMingmin**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by jbonofre** [comment]
R: @jbonofre

File: sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java [hunk 1]
```diff
@@ -279,8 +285,8 @@ Connection getConnection() throws Exception {
 
     public Read<T> withStatementPrepator(StatementPreparator statementPreparator) {
```
Comment (jbonofre) [inline sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java]: Actually, there's an issue there. It should be `withStatementPreparator()`. Can you fix that as well ?

**Comment by XuMingmin** [comment]
@jbonofre any pending change required for this PR?

**Comment by jbonofre** [comment]
Let me do a new round. Thanks !

**Comment by jbonofre** [comment]
Retest this please

**Comment by jbonofre** [comment]
Jenkins failure is not related but I requested a new build anyway.

**Comment by jbonofre** [comment]
Got a network issue, gonna merge asap.

**Comment by XuMingmin** [comment]
Thanks @jbonofre ,

**Pull request closed without merging**